### PR TITLE
Merge updates for Japanese translation.

### PR DESCRIPTION
**Pull request by ohtaman**

### CODE REVIEW

**Comment by ohtaman** [comment]
Sorry, this PR is sent by mistake

File: requirements.in [hunk 1]
```diff
@@ -17,5 +17,12 @@ plyvel
 pyyaml
 sklearn
 spotipy
-tensorflow
+tf-nightly-gpu-2.0-preview
```
Comment (DOsinga) [inline requirements.in]: Do we need this? It seems by default we probably want to go with the default tensorflow?

**Review by DOsinga — COMMENTED** [review]
This is great! And a little embarrassing that this wasn't fixed before, so let's merge this.

**Comment by DOsinga** [comment]
Sorry about that, inadvertently closed this.

**Comment by ohtaman** [comment]
Sorry, this PR my mistake and so I closed this by myself.

In fact, now I'm trying to update whole samples for the Japanese version of your book.
Is there any good way to collaborate with this original repository?
Now, we are working to

- support TensorFlow 2.0.
- remove/update some redundant codes.
- maybe remove some code which uses outdated libraries or methods.
   - and it may be possible to add new more modern codes.
- translate markdowns to Japanese.

I believe some of the above (maybe except the last one) are worthful also for this original repository.

**Comment by DOsinga** [comment]
Yes, that sounds great!

Why don't you send me a PR with the English only changes and then we just create a separate repository for the Japanese version? I don't think notebooks support dual language yet :)

I'm at douwe.osinga@gmail.com if you want to email directly. Happy to chat on whatever medium.

**Comment by DOsinga** [comment]
Hey!

How is this going? I pushed some updates and fixes partly based on your work. Happy to incorporate anything else that is reasonable. Let me know!

Douwe

**Comment by ohtaman** [comment]
Hi! Now we are working on translation towards publishing, hopefully in September. 

Since it is diffucult to manage *.ipynb on git (difficult to see diff, difficult to resolve conflicts, ... ), I want to keep PRs simple. Thus I'm thinking to close this PR and create other branches and PRs per .ipynb file. Is it OK?

**Comment by ohtaman** [comment]
FYI: https://app.reviewnb.com/ maybe help to treat  ipynb on github. I use this tool for tensorflow/docs translation.

**Pull request closed without merging**